### ISSUE TITLE
E8a/E9a - Issue with heading 3

### ISSUE THREAD

**Issue by jpinsent** [brharp/hjckrrh#149]
The date displaying the current week/month (eg. "Week of Sunday, August 9, 2015") is an h3. The only h2 above it is for Primary Tabs. While I'm not sure that this directly fails any criterion, it is misleading for users with a screen reader. The date should be a heading 2. 

This issue may be related to #142 

![display](https://cloud.githubusercontent.com/assets/12450480/9253650/fd2f5ab2-41ac-11e5-8023-7c967ccfdfcd.PNG)

**Comment by mmafe** [comment]
Same issue as #138 - fixed as part of hotfix_20150807.3.0

**Comment by jpinsent** [comment]
Confirmed to be working; the date is now a heading 2. This issue has been resolved.
